client/gui/CAnimation.h

```diff
@@ -161,17 +162,22 @@ class CFadeAnimation
 public:
 	enum class EMode
 	{
-		NONE, IN, OUT
+		NONE,
+		IN,
+		OUT,

```

Comment:
enumerating is single line is not allowed?

Reply:
Currently - no. Do we need it?

Reply:
Why not allowing? Take a look at ProtocolConsts from lobby.h
It would be awful.
Also, what is practical benefit from it?

Reply:
What's wrong with ProtocolConsts? What's so special about it?

Also, what is practical benefit of code formatting?

Reply:
Practical benefit from code formatting is to simplify code reading. 
Simplicity of code reading slightly correlates with amount of text on developer's screen. Long lines - bad, many lines - also bad.
Thus, should be balance between line width and amount of lines but modern IDEs do text wrapping, but they do nothing with long files. IMO, there is nothing wrong in having NONE, INE, OUT in one line

Reply:
don't see any sense in having enum values on one line